python/apps/taiga/tests/integration/taiga/projects/projects/test_api.py

```diff
@@ -172,36 +183,41 @@ async def test_get_project_being_no_project_member(client):
     assert response.status_code == status.HTTP_403_FORBIDDEN, response.text
 
 
-async def test_get_project_being_anonymous(client):
+async def test_get_project_403_forbidden_being_anonymous(client):
     project = await f.create_project()
 
     response = client.get(f"/projects/{project.b64id}")
     assert response.status_code == status.HTTP_403_FORBIDDEN, response.text
 
 
-async def test_get_project_not_found_error(client):
+async def test_get_project_404_not_found_project_b64id(client):
     user = await f.create_user()
-    non_existent_id = "xxxxxxxxxxxxxxxxxxxxxx"
-
     client.login(user)
-    response = client.get(f"/projects/{non_existent_id}")
+    response = client.get(f"/projects/{NOT_EXISTING_B64ID}")
     assert response.status_code == status.HTTP_404_NOT_FOUND, response.text
 
 
+async def test_get_project_424_unprocessable_project_b64id(client):
```

Comment:
This tests returns status.HTTP_422_UNPROCESSABLE_ENTITY

Reply:
Why I set a 422? My mistake, I'll change it.